### PR TITLE
fix: fix message

### PR DESCRIPTION
**Pull request by britaniar**
### Description of your changes

Fixes #

I have: capitalize message and label as template

- [x] Run `make reviewable` to ensure this PR is ready for review.

### Special notes for your reviewer

Addresses comments on #825

### CODE REVIEW

File: pkg/scheduler/framework/framework.go [hunk 1]
```diff
@@ -39,10 +39,10 @@ const (
 	eventRecorderNameTemplate = "scheduler-framework-%s"
 
 	// The reasons to use for scheduling decisions.
-	pickedByPolicyReason                  = "picked by scheduling policy"
-	pickFixedInvalidClusterReasonTemplate = "cluster \"%s\" is not eligible for resource placement yet: %s"
-	pickFixedNotFoundClusterReason        = "specified cluster \"%s\" is not found"
-	notPickedByScoreReason                = "cluster \"%s\" does not score high enough (affinity score: %d, topology spread score: %d)"
+	pickedByPolicyReason                   = "picked by scheduling policy"
```
Comment (ryanzhang-oss) [inline pkg/scheduler/framework/framework.go:42]: why is this message not start with capital letter?

Comment (britaniar) [reply pkg/scheduler/framework/framework.go:42]: It is used at the of [resourceScheduleSucceededMessageFormat ](https://github.com/Azure/fleet/blob/754df442ec85dbc599c22c62a1201960e02836f7/pkg/scheduler/framework/frameworkutils.go#L26C2-L26C114) and [resourceScheduleSucceededWithScoreMessageFormat](https://github.com/Azure/fleet/blob/754df442ec85dbc599c22c62a1201960e02836f7/pkg/scheduler/framework/frameworkutils.go#L27). I can delete and just add the statement to the end of these.

Comment (ryanzhang-oss) [reply pkg/scheduler/framework/framework.go:42]: next PR then. We should group all the reason in one file, it's scattered now.

**Review by ryanzhang-oss — APPROVED** [review]

File: pkg/scheduler/framework/framework.go (hunk 1 above)
Comment (britaniar) [reply pkg/scheduler/framework/framework.go:42]: Addressed on #845

**Pull request merged**